docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Running the Airy Core Platform in production
+sidebar_label: Production
+---
+
+This document provides our recommendations on how to run the Airy Core Platform in production environments. If you are not familiar with the architecture of the system, we suggest you read the [Architecture](architecture.md) document before proceeding.
+
+
+## Requirements 
+
+The `Airy apps` are the internal and exposed services, which comprise the `Airy Core Platform`. They run as docker containers and require access to several base services to be in place before they can be started:
+- `Kafka cluster`: Kafka, Zookeeper and the Confluent Schema registry. These three services comprise the Kafka store. They are the default storage system of the Airy Core Platform. Kafka requires Zookeeper to  work. The Confluent Schema registry facilitates Avro typed data pipelines. All our Kafka based applications require the registry to work.
+- `PostgreSQL`: Where we store authentication data.
+- `Redis`: The queuing system used by our webhook system relies on Redis.
+
+### Kafka cluster
+
+The Kafka store requires multiple Kafka brokers, zookeepers, and Confluent Schema registry servers. We recommend using at least five Kafka brokers and  set the replication factor of _all_ topics to `3`.  So that even if two brokers would become unavailable at the same time, the system would still function. 
+
+We also recommend running at least three zookeepers and two instances of the Confluent Schema registry.
+
+If you decide to run the Kafka cluster on Kubernetes, we recommend running Kafka and Zookeeper as StatefulSet workloads, for persistent storage. The Confluent Schema registry can be run as a Deployment, as it is stateless.
+
+Apart from running your own kafka cluster, a managed version can also be used.
+
+The kafka cluster is usually started in the following order:
+
+- Start the `ZooKeeper` nodes. The ZooKeeper nodes need to have the ports 2888 and 3888 open between them and the port 2181 open to the Kafka servers. Note down the hostnames of the zookeeper servers because they are required for the configuration of the Kafka brokers.
```

Comment:
```suggestion
- Start the `ZooKeeper` nodes. The ZooKeeper nodes need to have the ports `2888` and `3888` open between them and the port `2181` open to the Kafka servers. Note down the hostnames of the ZooKeeper servers because they are required for the configuration of the Kafka brokers.
```